truffle.js

```diff
@@ -91,7 +91,9 @@ module.exports = {
       network_id: "*",
     },
     xdai: {
-      url: "https://xdai-archive.blockscout.com/",
```

Comment:
I actually quite value having `--network xdai` as something that works in the default repo if I have to check something on the contract level.